test/01_cond_field/process_simple.py

```diff
@@ -319,14 +320,15 @@ def all_collect(self, sampler):
 
     def calculate_moments(self, sampler):
         """
+        @TODO: refactor - use quantity
         Calculate moments through the mlmc.QuantityEstimate
         :param sampler: mlmc.Sampler
         :return: None
         """
         # Simple moment evaluation
-        moments_fn = self.set_moments(sampler.sample_storage)
+        moments_fn = self.set_moments(sampler._sample_storage)
 
-        q_estimator = QuantityEstimate(sample_storage=sampler.sample_storage, moments_fn=moments_fn,
+        q_estimator = QuantityEstimate(sample_storage=sampler._sample_storage, moments_fn=moments_fn,
```

Comment:
However `quantity_estimate.py` is deleted in this very PR.
And the import is removed.
We have to manually check and fix this real test after all changes.

Reply:
I think `calculate_method` is no longer in use in the end. Either way I use this file a lot and eventually, everything works.